README, updating document (#159)
omise/omise-woocommerce: README.md

#### Enable payment methods

> 1. After setting up Omise Account, at the Omise Settings page, **"Payment Methods"** section, there will be a number of available payment methods shown on the list.

Reviewer: ```suggestion
1. After setting up your Omise Account, there will be a number of available payment methods shown in the **"Payment Methods"** section of the Omise Settings page.
```

Once done, those payment methods will be shown at the store's checkout page.
  ![screen shot 2560-07-26 at 8 13 55 pm](https://user-images.githubusercontent.com/2154669/28622536-030403e2-723f-11e7-8a93-a06e65e350d3.png)

### Available Payment Methods

Payment methods that are available on the store will be determined based on the country that merchants are registered.
The following is the list of payment methods that are supported in each country.

#### 🇯🇵 Japan
Credit / Debit Card

#### 🇸🇬 Singapore
Credit / Debit Card, and PayNow

#### 🇹🇭 Thailand

> Alipay, Bill Payment: Tesco, Credit / Debit Card, Installment, Internet Banking, and TrueMoney Wallet.

Reviewer 2: `Bill Payment: Tesco` -> `Bill Payment: Tesco Lotus` ?

> Alipay, Bill Payment: Tesco, Credit / Debit Card, Installment, Internet Banking, and TrueMoney Wallet.

Reviewer: ```suggestion
Alipay, Bill Payment: Tesco Lotus, Credit / Debit Card, Installment, Internet Banking, and TrueMoney Wallet.
```
